src/System.Security.Cryptography.ProtectedData/tests/ProtectedDataTests.cs

```diff
@@ -15,6 +15,7 @@ namespace System.Security.Cryptography.ProtectedDataTests
     public static class ProtectedDataTests
     {
         [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
```

Comment:
*If* we decide to add this, it can be added to the class rather than adding it to each Fact.

Reply:
Oh, actually I didn't think about it. Let me try this option.